website/src/graphql/dailp/index.ts

```diff
@@ -425,8 +484,55 @@ export type DeleteContributorAttribution = {
   readonly documentId: Scalars["UUID"]
 }
 
+/** 
+ *  Represents the format of a citation. 
+ *  Used to map specific format to their more general citation format.
+ */
+export enum DocCitationFormat =
```

Comment:
I would recommend changing this enum to all caps since the rest of the enums in this file look like that. So like 
`Website = "WEBSITE"`